test/e2e/query_test.go

```diff
@@ -167,6 +170,53 @@ func TestQuery(t *testing.T) {
 	})
 }
 
+func TestQueryRoutePrefix(t *testing.T) {
+	t.Parallel()
+
+	s, err := e2e.NewScenario("e2e_test_query_route_prefix")
+	testutil.Ok(t, err)
+	defer s.Close()
+
+	q, err := e2ethanos.NewQuerier(
+		s.SharedDir(), "1",
+		nil,
+		nil,
+		nil,
+		"test",
+	)
+	testutil.Ok(t, err)
+	testutil.Ok(t, s.StartAndWaitReady(q))
+
+	ctx, cancel := chromedp.NewContext(context.Background())
+	defer cancel()
+
+	var networkErrors []string
+
+	// Listen for failed network requests and push them to an array.
+	chromedp.ListenTarget(ctx, func(ev interface{}) {
+		switch ev := ev.(type) {
+		case *network.EventLoadingFailed:
+			networkErrors = append(networkErrors, ev.ErrorText)
+		}
+	})
+
+	err = chromedp.Run(ctx,
+		network.Enable(),
+		chromedp.Navigate("http://"+q.HTTPEndpoint()+"/test/graph"),
+		chromedp.WaitVisible(`body`),
+	)
+	testutil.Ok(t, err)
+
+	err = func() error {
+		if len(networkErrors) > 0 {
+			return fmt.Errorf("some network requests failed")
```

Comment:
if we just return this error, then we won't know _why_ chromedp is failing. How about concatenating the error strings?
```go
return fmt.Errorf("some network requests failed: %s", strings.Join(networkErrors, "; "))
```

Reply:
Got it. I'll push the changes.